model/src/main/java/com/basistech/rosette/dm/BaseAttribute.java

```diff
@@ -28,9 +28,13 @@
 import java.util.Set;
 
 /**
- * Base class for attributes that annotate text.  Each attribute contains
- * an extended properties map of String to Object to hold optional
- * "user-defined" elements.
+ * Base class for attributes that annotate text.
+ * <p>
+ * Each attribute contains an extended properties map of {@code String}
+ * to {@code Object} to hold optional "user-defined" elements. An
```

Comment:
This is describing the implementation details. Wouldn't better something like "Each attribute may contain `extended properties` which users may add additional properties. An extended property key should..."

Reply:
The documentation already described it as a “map of String to Object”; I just reformatted it. `getExtendedProperties` is public and has the return type `Map<String, Object>`. So I think this is an API guarantee, not an implementation detail.